src/Tribe/Admin/Notices.php

```diff
@@ -137,13 +212,13 @@ public function maybe_display_plus_commerce_notice() {
 		$plus = sprintf(
 			'<a target="_blank" rel="noopener nofollow" href="%s">%s</a>',
 			esc_attr( $plus_link ),
-			esc_html( 'Event Tickets Plus', 'event-tickets' )
+			esc_html__( 'Event Tickets Plus', 'event-tickets' )
 		);
 
-		$plus_commerce_providers = array(
-			esc_html( 'WooCommerce', 'event-tickets' )            => 'woocommerce/woocommerce.php',
-			esc_html( 'Easy Digital Downloads', 'event-tickets' ) => 'easy-digital-downloads/easy-digital-downloads.php',
-		);
+		$plus_commerce_providers = [
+			esc_html__( 'WooCommerce', 'event-tickets' )            => 'woocommerce/woocommerce.php',
```

Comment:
:warning: Array double arrow not aligned correctly; expected 1 space(s) between ")" and double arrow, but found 12.

<sub>hash: 32ee214b91655c059bf559b1caaeea53</sub>